src/main.rs

```diff
@@ -347,14 +351,22 @@ fn read_tasks() -> Result<Vec<Group>> {
 fn create_process(task: &Task) -> Result<Child> {
     let current_dir = current_dir()?;
     let working_dir = task.working_dir.as_ref().unwrap_or(&current_dir);
-    let child = Command::new("sh")
-        .args(["-c", &format!("exec {}", task.cmd)])
+    let mut child = Command::new("sh");
+        child.args(["-c", &format!("exec {}", task.cmd)])
         .current_dir(working_dir)
         .stdin(Stdio::inherit())
         .stdout(Stdio::inherit())
-        .stderr(Stdio::inherit())
-        .spawn()?;
-    Ok(child)
+        .stderr(Stdio::inherit());
+
+    if task.clear_env {
+        child.env_clear();
+    }
+
+    for (key, value) in &task.env {
+        child.env(key.clone(), value.clone());
+    }
```

Comment:
You can be a little bit more expressive by using `Command::envs()` method. It can accept reference to any type that can be converted to `Iterator<(String, String)>` which `HashMap<String, String>` is.